### PR TITLE
Fixes #14414: Create migration script for parent ticket

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/14414

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-upgrade [hunk 1]
```diff
@@ -721,6 +722,13 @@ upgrade_database() {
     echo " Done"
   fi
 
+  # - 4.1.21: Drop constraint rulesgroupjoin_pkey
+  RES=$(${PSQL} -t -d ${SQL_DATABASE} -c "select count(*) from information_schema.constraint_column_usage where table_name = 'rulesgroupjoin'")
+  if [ $RES -ne 0 ]; then
+    echo -n "INFO: Dropping constraint 'rulesgroupjoin'"
+    $${PSQL} -d ${SQL_DATABASE} -f ${RUDDER_UPGRADE_TOOLS}/dbMigration-4.1.x-4.1.21-drop-rulesgroupjoin-constraint.sql > /dev/null
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/rudder-upgrade]: $$ ?

**Comment by peckpeck** [comment]
The package has changed in master, you will need a second patch for master

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**